### ISSUE TITLE
Calendar not accepting weekday labels

### ISSUE THREAD

**Issue by NullCodex** [patientslikeme/react-calendar-heatmap#74]
Currently, it looks like the calendar is not accepting any new labels? Additionally the css looks off? Overlapping with the heat map itself.

I'm calling it with:
```
        <CalendarHeatmap
          endDate={new Date()}
          startDate={beginningDate}
          values={data}
          showWeekdayLabels={true}
          weekdayLabels={['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat']}
        />
```

The displayed image looks like:
<img width="360" alt="screen shot 2018-01-29 at 7 52 09 pm" src="https://user-images.githubusercontent.com/7586288/35542426-f2b0c002-052d-11e8-993f-6c5fd7594333.png">

**Comment by NullCodex** [comment]
Hi any updates?

**Comment by iqnivek** [comment]
Hi @NullCodex! Sorry about the delay in responding.

It turns out `weekdayLabels` specifies the strings used for the labels, but only the odd-indexed labels are shown. I think this behavior is pretty confusing, and I'd be up for changing it - my initial thought would be changing weekdayLabels to do what you'd expect (render all the specified labels), and we can then remove showWeekdayLabels. In the meantime, if you fork or vendor the library and remove [this conditional](https://github.com/patientslikeme/react-calendar-heatmap/blob/master/src/index.js#L300), you'll be able to get the behavior you're looking for.

I'm not sure sure what's up with the overlapping. Could you fork [this codesandbox](https://codesandbox.io/s/73mk9wlyx) with your CSS and see if you can reproduce it? The default styles seem to be okay and not overlapping, so my guess is that it might be other CSS that's affecting things, but it's hard to say without a test case.

**Comment by cyclops24** [comment]
@kevinsqi Any plan to fix this?

**Comment by kevinsqi** [comment]
Yes, PRs would be welcome. I'd like to work on this as well, if I have time.

**Comment by rastegars** [comment]
For me, it looks like this:

<img width="550" alt="Screen Shot 2020-01-12 at 8 23 27 PM" src="https://user-images.githubusercontent.com/3218684/72222491-654c4180-357a-11ea-9425-c8f79d34535e.png">

I fixed the overlapping problem temporarily, by reseting translation value for the X-axis to zero:
 
```
const weekdayLabels = document.getElementsByClassName('react-calendar-heatmap-weekday- 
  labels');
weekdayLabels.forEach((element) => {
  element.setAttribute('transform', 'translate(0, 14)');
});
```

**Comment by kunaldongre24** [comment]
Hey there, How can I show dates of the boxes onMoueOver? 
Even the boxes which has null values.
Please help
